manuscript/03.results.md

```diff
@@ -46,7 +46,7 @@ More specifically, one of the benchmarking studies we performed was with data ma
 
 Cells of even very distinct molecular states can have partly overlapping phenotypes due to non-heritable variation. Therefore, we sought to evaluate how different two states would need to be for us to accurately identify them as distinct (Fig. {@fig:wass}a). We varied the G1 phase duration of two states from identical to very distinct (Wasserstein distance greater than 70) (Fig. {@fig:wass}b) and quantified the accuracy of our model. While the phenotypic observation of a given state had to be different for our model to accurately assign cells, even moderately overlapping phenotypes could be distinguished by using the lineage relationships of cells (Fig. {@fig:wass}c). As a baseline comparison, we used K-means clustering to cluster cells based on their phenotype without using their lineage relationships (Fig. {@fig:wass}c). A tHMM consistently outperformed this approach.  The model performance in censored and uncensored populations were similar (Fig. @fig:wass1, @fig:wass2). This shows that the heritability of a cell state can help to more accurately identify cells with partially overlapping phenotypes.
 
-![**Model performance versus the Wasserstein distance between states.** (a) Cartoon of how two states can vary in their phenotypic similarity. On the left, cells might be virtually indistinguishable (here based on shape). On the right, they might be so different that looking at one cell is sufficient to identify its state. (b) The distribution of G1 duration is varied in state 1 (blue) while the other state is kept constant. (c) State assignment accuracy versus the Wasserstein distance between state phenotypes. Each point represents the accuracy of state assignment for a lineage created by a set of parameters that yield the shown Wasserstein distance between the two state distributions. Either a K-means model was used with the phenotypes of each individual cell (orange) or the full tHMM model was applied (blue). The solid lines show a Lowess trendline of the model accuracy.](figure6.svg){#fig:wass}
```

Comment:
It seems like the only fix here is replacing left/right with top/bottom. The new text is wrong; difference/similarity is judged between states, not within them.